docs/topics/practices.rst

```diff
@@ -209,6 +236,8 @@ Same example but running the spiders sequentially by chaining the deferreds:
         reactor.stop()
 
 
+    from twisted.internet import reactor
+
     crawl()
```

Comment:
I think install_reactor() is called here so the import is probably still too early, but I'm not 100% sure.

Reply:
Is not called because we are using `CrawlerRunner`, but yes if we are using different `TWISTED_REACTOR` we need to call `install_reactor` before importing `reactor` here.